app/src/main/java/src/Shape.java

```diff
@@ -4,27 +4,58 @@
     private int xPosition;
     private int yPosition;
 
+    public Shape(int xPos, int yPos) {
+        xPosition = xPos;
+        yPosition = yPos;
+    }
+
     @Override
     public abstract int[][] draw();
 
+    /**
+     * Adds one to yPosition.
+     * In order to represent to move right
+     */
     @Override
     public void moveRight() {
-
+        yPosition += 1;
     }
 
+    /**
+     * Adds one to yPosition.
```

Comment:
Description on both methods is very similar. 
Maybe you could try something like:
Increase yposition by one
Decrease yposition by one

Reply:
Thanks for suggestions.